docker/Dockerfile

```diff
@@ -1,15 +1,13 @@
 # python 3
 FROM continuumio/miniconda3
 
-MAINTAINER Attawit Kittikrairit
-
 # set working directory
 WORKDIR /usr/src/app
 
 # clone from repo
-RUN git clone https://github.com/e-mission/e-mission-server.git .
+RUN git clone https://github.com/alvinalexander/e-mission-server.git .
```

Comment:
leave this as e-mission in the checked in code. You will soon be merging the generation code into the server repository as well!

Reply:
This is unresolved.